VadeSecure/vade_secure_m365/_meta/fields.yml

```diff
@@ -0,0 +1,83 @@
+vadesecure.folder:
+    description: The name of the folder in the end user's mailbox if the message has been moved by Vade.
+    name: vadesecure.folder
+    short: vadesecure.folder
+    type: keyword
+vadesecure.from_header:
+    description: 'The email address declared in the From: header of the message, which may differ from the address used in the SMTP MAIL FROM command.'
```

Comment:
```suggestion
    description: The email address declared in the From: header of the message, which may differ from the address used in the SMTP MAIL FROM command.
```

Reply:
Quotes are needed as the value contains `:` character